src/oop/oop2.py

```diff
@@ -1,23 +1,26 @@
-# To the GroundVehicle class, add method drive() that returns "vroooom".
-#
-# Also change it so the num_wheels defaults to 4 if not specified when the
-# object is constructed.
 
-class GroundVehicle():
-    def __init__(self, num_wheels):
-        self.num_wheels = num_wheels
+# class GroundVehicle
 
-    # TODO
+class GroundVehicle():
+    def __init__(self, num_wheels=None):
+        if num_wheels == None:
+            self.num_wheels = 4
+        else:
+            self.num_wheels = num_wheels
```

Comment:
you can do the same thing here a bit cleaner...
```
def __init__(self, num_wheels=4)
    self.num_wheels  = num_wheels
```
So we'll default the num_wheels to 4 if nothing is passed in, otherwise we'll set it to whatever we do pass in for that parameter.